packages/remix-forms/src/createForm.tsx

```diff
@@ -302,7 +302,20 @@ function createForm({
       } as Field<SchemaType>
     }
 
-    const globalErrors = errors?._global
+    const deepHiddenFieldsErrors = hiddenFields?.map((hiddenField) => {
```

Comment:
I'd like to extract this elsewhere, but it's also coupled with the `fieldErrors` function, what do you ahh think?

Reply:
I don't think we gain much by extracting elsewhere, but if you can turn everything from lines 305 to 318 into a function that you can apply to `errors?._global` and get the const `globalErrors` it would be easier to reason about the code. 

I admit it also tickles my functional fancy :P

Reply:
Thanks for the idea